storage/ndb/src/kernel/blocks/dbtc/DbtcMain.cpp

```diff
@@ -15843,6 +15843,8 @@ void Dbtc::execSCAN_TABREQ(Signal *signal) {
   scanptr.p->m_scan_dist_key = scanTabReq->distributionKey;
   scanptr.p->m_scan_dist_key_flag = ScanTabReq::getDistributionKeyFlag(ri);
 
+  scanptr.p->m_ttl_purge_window_size = scanTabReq->ttlPurgeWindowSize;
```

Comment:
I don't see any upgrade code here, thus if an old NDB API sends a SCAN_TABREQ the value of ttl_purge_window_size is undefined which from what I can see will cause issues.